test/idle-timeout.ts

```diff
@@ -36,8 +36,8 @@ test('idle timeout will let go of threads early', async ({ is }) => {
   is(pool.threads.length, 1);
 
   const secondTasks = [
-    pool.runTask(busyLoop(100) + getThreadId),
-    pool.runTask(busyLoop(100) + getThreadId)
+    pool.runTask(busyLoop(500) + getThreadId),
+    pool.runTask(busyLoop(500) + getThreadId)
```

Comment:
Saw this previously but hadn't asked... would a sync sleep using Atomics.wait() work here also? Or does it need to busy loop the cpu?

Reply:
Yeah, I think `Atomics.wait` would work too. I’ll try to refactor this test anyway since it still seems to be failing in CI